src/Chainweb/Pact/Backend/Compaction.hs

```diff
@@ -456,7 +456,7 @@ dropNewTables bh = do
 --   properly pruning RocksDB.
 compactSystemTables :: BlockHeight -> CompactM ()
 compactSystemTables bh = do
-  let systemTables = ["BlockHistory", "VersionedTableMutation", "VersionedTableCreation"]
+  let systemTables = ["BlockHistory", "VersionedTableMutation"]
```

Comment:
the `column` variable is no longer needed - for every table in the loop, the column name is just "blockheight".

Reply:
ping on this